plugins/defaultplugins/l3plugin/model/l3/l3.proto

```diff
@@ -3,17 +3,17 @@ syntax = "proto3";
 package l3;
 
 message StaticRoutes {
-    message Ip {
+    message Route {
         uint32 vrf_id = 1;                   /* VRF identifier */
```

Comment:
this field is basicly optional since you have this information in the key. To avoid inconsitency between key and this field you:
- either remove this field
- or overwrite it in the implementation by the value that you parse from key

I think the second alternative is used in existing implementation (please check ifaces and bds)